hedera-node/test-clients/src/rcdiff/java/com/hedera/services/rcdiff/RcDiff.java

```diff
@@ -182,20 +186,17 @@ private String readableDiff(@NonNull final DifferingEntries diff) {
         if (firstEncounteredDifference == CONSENSUS_TIME_MISMATCH) {
             sb.append("➡️  Expected ")
                     .append(Objects.requireNonNull(diff.firstEntry()).consensusTime())
-                    .append(" but was ")
+                    .append("\n\n")
+                    .append("➡️ but was ")
                     .append(Objects.requireNonNull(diff.secondEntry()).consensusTime());
         } else if (firstEncounteredDifference == TRANSACTION_RECORD_MISMATCH
                 || firstEncounteredDifference == TRANSACTION_MISMATCH) {
             sb.append("\nFor body,\n")
                     .append(Objects.requireNonNull(diff.firstEntry()).body());
-            sb.append("➡️  Expected Receipt ")
-                    .append(Objects.requireNonNull(diff.firstEntry())
-                            .transactionRecord()
-                            .getReceipt())
```

Comment:
Was it intentional to drop getReceipt() from here and below?

Reply:
yes it will include both and not just the reciept

Reply:
The record include the receipt